src/generators/app/templates/src/server/i18n.js

```diff
@@ -0,0 +1,34 @@
+import i18n from 'i18next';
+import Backend from 'i18next-node-fs-backend';
+import { LanguageDetector } from 'i18next-express-middleware';
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .init({
+    detection: {
+      order: ['cookie'],
```

Comment:
Let's use `querystring`. I think this is the most transparent option for the end user.